subprojects/core/src/test/groovy/org/gradle/internal/operations/notify/BuildOperationNotificationBridgeTest.groovy

```diff
@@ -43,7 +43,7 @@ class BuildOperationNotificationBridgeTest extends Specification {
         buildOperationListenerManager = Mock(BuildOperationListenerManager)
 
         when:
-        def bridge = bridge()
+        def bridge = createOrGetBridge()
```

Comment:
⭕ Just a nitpick, but `getOrCreate` sounds better.

Reply:
Handled in https://github.com/gradle/gradle/pull/15582